tests/Formatter/DEFormatterTest.php

```diff
@@ -36,6 +36,51 @@ public function providerFormat() : array
             ['ABCD', null],
             ['ABCDE', null],
             ['ABCDEF', null],
+
+            // invalid
+            ['010101', null],
+            ['000000', null],
+            ['(123456)', null],
+            ['!23456', null],
```

Comment:
The data passed to `CountryPostcodeFormatter::format()` can only contain uppercase alphanumeric characters, so testing with other characters is redundant here.

(we may revisit this in #17 though!)